packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -135,8 +135,11 @@ export function Autocomplete({
   }
 
   async function updateSearch() {
-    const opts = await getOptions(inputText);
-    setOptions(mapToOptions(opts));
+    const opts: AnyOption[] = await getOptions(inputText);
+    const filteredOpts = opts.filter((o: AnyOption) =>
```

Comment:
A preference here would be to use a variable name that is more descriptive. `o`, while works, doesn't tell us what we are looking at. A name like `option` may be better in this case

Reply:
Makes sense 👍 
Changed the naming a bit, hope it's better now.